assets/js/type-defs/contexts.js

```diff
@@ -212,17 +213,17 @@
  * @property {string}                  redirectUrl        This is the url that
  *                                                        checkout will redirect
  *                                                        to when it's ready.
- * @property {function()}              onCheckoutCompleteSuccess Used to register a
+ * @property {function(function())}    onCheckoutCompleteSuccess Used to register a
```

Comment:
What does this mean?

Reply:
It means that the "type" for this value is a function that receives a function as it's argument.